packages/helpers/qrcode-modal/src/browser/registry.ts

```diff
@@ -42,6 +42,48 @@ const officialAlgorandWallet: IAppEntry = {
   },
 };
 
+const fireblocksAlgorandWallet: IAppEntry = {
+  // hex encoding of 32 random bytes
+  id: "802bd272e62c1d7754b136e6398d4c4f6bac08744a59ebbdcd70f23fbbf7d3c0",
+  name: "Fireblocks",
+  homepage: "https://www.fireblocks.com",
+  chains: [
+    // need to change once a standard emerges, perhaps https://github.com/ChainAgnostic/CAIPs/pull/55
+    "algorand",
+  ],
+  // randomly generated UUID
+  image_id: "12024107-3dac-499e-9c69-5466a57722fd",
+  image_url: {
+    sm: "https://console.fireblocks.io/static/fireblocks-icon.png",
+    md: "https://console.fireblocks.io/static/fireblocks-icon.png",
+    lg: "https://console.fireblocks.io/static/fireblocks-icon.png",
+  },
+  app: {
+    browser: "",
+    ios: "https://apps.apple.com/us/app/fireblocks/id1439296596",
+    android: "https://play.google.com/store/apps/details?id=com.fireblocks.client",
+    mac: "",
+    windows: "",
+    linux: "",
+  },
+  mobile: {
+    native: "",
+    universal: "",
```

Comment:
Does Fireblocks not have a specific URI prefix for iOS, as described here: https://docs.walletconnect.com/mobile-linking#for-ios?

Without this information, I'm not sure this entry would have the right behavior.

Reply:
currently we don't have a specific scheme handler on iOS, I'll see if we can add one

Reply:
we implemented a custom scheme handler on our iOS app: "fireblocks-wc:" still not rolled out yet.

Reply:
update: the new version of the Fireblocks iOS app with the custom scheme was published to the App Store and should be rolled out to users gradually in the following week.

Reply:
Great, thank you! I will merge this PR once you've signed the CLA, as that is the only blocker now.

Reply:
done, thanks!

Reply:
hey @jasonpaulos you can go ahead and merge this